definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -3177,6 +3177,13 @@
     "UserRole": "caseworker-divorce-financialremedy-courtadmin",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "scannedDocuments",
+    "UserRole": "caseworker-divorce-financialremedy-judiciary",
+    "CRUD": "R"
+  },
```

Comment:
Ticket only mentions consented. Are we sure we need to add for contested?

Reply:
Good point, I assumed that it should be the same for both, I'll ask on the ticket for confirmation

Reply:
Lucy is fine with both being done